themes/default/content/blog/pulumi-release-notes-66/index.md

```diff
@@ -0,0 +1,113 @@
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/99)
+
+### Support `pulumi import` for Kubernetes CRDs
+
+We have added `pulumi import` support for Kubernetes CustomResourceDefiniton (CRD). Now the spec of a CRD will be imported during `pulumi import`. The same fix improves input generation for other Kubernetes resources as well, providing significantly better fidelity in covering inputs for existing resources.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes/issues/1410)
+
+### Various improvements to Helm `Release`
+
+This milestone we spent some time making improvements to the Helm `Release` support. Of particular note are the ability to import existing Helm releases installed via the Helm command line into Pulumi (#1818 ) and the ability to supply Helm values through YAML files (#1828). In addition, we have made a variety of bug fixes this iteration to make Helm Release a more robust option to use for your Kubernetes environment.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes/pull/1818) and [this GitHub issue](https://github.com/pulumi/pulumi-kubernetes/pull/1828).
+
+## Pulumi CLI and core technologies
+
+### Support using native ES modules as Pulumi scripts
+
+Native ECMAScript module (ESM) support has been added for the Node.js SDK. Pulumi users can now use Pulumi in projects with "type": "module" configured.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/7764)
+
+### Support packages with plugins hosted in any third-party location
+
+Pulumi Packages can now host their plugins anywhere (like GitHub releases) instead of needing to be published by Pulumi. We now detect any dependency that contains pulumi-plugin.json and treat it as a Pulumi Package, automatically downloading associated plugins as needed. To support this, the freshly generated Multi-Language Component (MLC) plugin will now include pulumi-plugin.json by default.
+
+Learn more in the following GitHub issues: [#8515](https://github.com/pulumi/pulumi/issues/8515), [#8516](https://github.com/pulumi/pulumi/issues/8516), [#8517](https://github.com/pulumi/pulumi/issues/8517), [#8530](https://github.com/pulumi/pulumi/issues/8530), [#8527](https://github.com/pulumi/pulumi/issues/8527) and [#8532](https://github.com/pulumi/pulumi/issues/8532).
+
+
+### State locking default enabled
+
+We [previously added](https://github.com/pulumi/pulumi/pull/2697) support for self-managed backend state locking behind the  PULUMI_SELF_MANAGED_STATE_LOCKING=1 flag. After positive feedback from users on this feature, we are making this the default when using a local or cloud backend such as Amazon S3, Google Cloud Storage and Azure Blob Storage. 
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/8565)
+
+## Pulumi Service & Pulumi.com
```

Comment:
I'd remove this:

```suggestion
## Pulumi Service
```

... since there isn't anything having to do with the website ("pulumi.com") here.